frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -313,14 +304,50 @@ If you'd like to change the order in which dictionaries are queried (and their r
 end
 
 function ReaderDictionary:onLookupWord(word, box, highlight, link)
+    logger.dbg("lookup word:", word, box)
```

Comment:
Since we're modifying this line anyway maybe add some prefix? `dict lookup word` and `dict stripped word` or something. (You know, for grepping.)